sksparse/cholmod.pyx

```diff
@@ -255,6 +288,16 @@ cdef void _error_handler(
     if status > 0:
         # Warning:
         warnings.warn(full_msg, CholmodWarning)
```

Comment:
We should also think about raising a `CholmodNotPositiveDefiniteError` error here if the status is `CHOLMOD_NOT_POSDEF` (1). For now just a warning is raised.